Makefile

```diff
@@ -15,7 +15,7 @@ $(tests): %.py : lint
 	coverage run -p --source=fusillade $*.py -v
 
 integration_test:
-	$(MAKE) FUS_TEST_MODE=integration test
+	source environment && $(MAKE) FUS_TEST_MODE=integration test
```

Comment:
nit, move the `source environment` to a  new rule target, then place the new `source` target as a prerequisite to the rules that require that action. 

Make it easier to read ha.

Reply:
I don't think make works that way since every line is a new shell, the environment variables wont be set. I'm not fluent in shell so please correct me if I'm wrong. This what I've come to understand from my research.

Reply:
https://stackoverflow.com/questions/7507810/how-to-source-a-script-in-a-makefile